packages/cache/README.md

```diff
@@ -4,7 +4,7 @@
 
 See ["Caching dependencies to speed up workflows"](https://help.github.com/github/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows) for how caching works.
 
-Note that GitHub will remove any cache entries that have not been accessed in over 7 days. There is no limit on the number of caches you can store, but the total size of all caches in a repository is limited to 5 GB. If you exceed this limit, GitHub will save your cache but will begin evicting caches until the total size is less than 5 GB.
+Note that GitHub will remove any cache entries that have not been accessed in over 7 days. There is no limit on the number of caches you can store, but the total size of all caches in a repository is limited to 10 GB. If you exceed this limit, GitHub will save your cache but will begin evicting caches until the total size is less than 10 GB.
```

Comment:
I know it's on the TODO, but just a friendly reminder to also update the public docs :smile: - https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy

Reply:
Noted. I have the PR ready. Will raise it as soon as this goes live.

Reply:
Linking the PR raised by @N-Usha for updating the doc - https://github.com/github/docs-internal/pull/22820/files

Reply:
:+1: